pxf/pxf-jdbc/src/test/resources/core-site.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The test/resource directory contains information about specific setup (hadoop & mysql), and should probably not be included here.